src/plugins/dam/components/uploader/FileList.jsx

```diff
@@ -32,22 +33,21 @@ class FileList extends React.Component {
     super(props);
 
     this.handleOnChange = this.handleOnChange.bind(this);
+    this.fileList = React.createRef();
+    this.ensureActiveItemVisible = this.ensureActiveItemVisible.bind(this);
```

Comment:
@digitalartcrew you can just throttle it here in the constructor, check out the example here https://medium.com/@markotroskot/debouncing-and-throttling-in-react-feb84d84bf28